src/main/flight/rpm_filter.c

```diff
@@ -173,6 +173,7 @@ FAST_CODE_NOINLINE void rpmFilterUpdate()
         if (motor < 4) {
             DEBUG_SET(DEBUG_RPM_FILTER, motor, motorFrequency[motor]);
         }
+        motorFrequency[motor] = erpmToHz * filteredMotorErpm[motor];
```

Comment:
What is the rationale for this change?

Reply:
The old way resulted in intermittent rpm values, only when there was a significant change.  This led to a great deal of jitter on the derivative of the RPS signal.  I discussed it with Thorsten and he recommended this change, which definitely improves matters.